code/src/App.js

```diff
@@ -1,9 +1,101 @@
 import React from 'react'
+import { useState } from 'react'
+import { Header } from './Header'
+import { Footer } from './Footer'
+import './App.css'
+import { format } from 'path'
 
 export const App = () => {
+  const [movie, setMovie] = useState("");
+  const [showResult, setShowResult] = useState(false);
+  const [songName, setSongName] = useState("");
+  const candyGroups = ["Gingerbread", "Chocolate", "Butterscotch"];
+  const [candyGroup, setCandyGroup] = useState();
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setShowResult(true);
+  };
```

Comment:
Would be nice with some error handeling so it doesn't always get true when you pressed the button (only when you have values)